packages/lib-react-components/src/ZooHeader/components/NavListItem/NavListItem.js

```diff
@@ -36,7 +36,14 @@ export default function NavListItem ({ label, url }) {
   )
 }
 
+NavListItem.defaultProps = {
+  color: '#B2B2B2',
```

Comment:
This colour should be added to the theme at some point, it's used in a couple of other places.